lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -185,6 +199,20 @@ mkJormungandrLayer mgr baseUrl = JormungandrLayer
                         (BlockId parentId)
                         (Just count)
                 left ErrGetDescendantsNetworkUnreachable <$> defaultHandler ctx x
+    , postSignedTx = \tx -> ExceptT $ do
+        let e0 = "?"
```

Comment:
I can't find where this was taken from?

Reply:
Sorry, for the confusion by including this commit. (I shouldn't have!) I copied it from `http-bridge` where `?` was an error message. I have not at all looked into what kind of errors jormungandr returns. Dropping it now.